other-docs/guides/automatic-updates.md

```diff
@@ -0,0 +1,34 @@
+# Automating Updates
+
+The recommended approach to automating updates to Altis is to use GitHub's Dependabot feature.
+
+The service monitors the dependencies in your project and creates automatic pull requests whenever there are updates. This will help you to keep your projects healthy and always running the latest patch release of all Altis modules.
+
+## Setting Up Dependabot
+
+To get started create a `dependabot.yml` file in your project root.
+
+The minimum recommended configuration for Altis is as follows:
+
+```yaml
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "composer"
+    # Look for `composer.json` and `composer.lock` files in the `root` directory
+    directory: "/"
+    # Check packagist registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
```

Comment:
Can this be live instead? AFAIK that's supported for Composer.

Reply:
So it's not in GitHub's documentation anywhere, feels a little risky but if we have some examples already working with this then 👍 

Updated.